roles/ansible_service_broker/templates/broker-user-auth.clusterrole.yaml.j2

```diff
@@ -0,0 +1,18 @@
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: asb-user-access
+  labels:
+{% if ansible_service_broker_sandbox_role == 'admin' %}
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+{% endif %}
+{%if ansible_service_broker_sandbox_role == 'edit' %}
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
rather than templating these blocks it might make more sense to just template the value of the label to "true" or "false"